### PR TITLE
feat: use GraphQL for all server-machine communication

### PR DESCRIPTION
**Pull request by siiky**
Related to https://github.com/lamassu/lamassu-machine/pull/773

### CODE REVIEW

File: lib/plugins.js [hunk 1]
```diff
@@ -365,12 +439,12 @@ function plugins (settings, deviceId) {
 
         const rate = rawRate.div(cashInCommission)
 
-        const lowBalanceMargin = new BN(1.05)
+        const lowBalanceMargin = new BN(0.95)
```
Comment (chaotixkilla) [inline lib/plugins.js:368]: Any specific reason for this change?

File: lib/routes.js [hunk 2]
```diff
@@ -56,10 +60,13 @@ app.use('/', pairingRoutes)
 app.use(findOperatorId)
 app.use(populateDeviceId)
 app.use(computeSchema)
-if (!devMode) app.use(authorize)
+app.use(authorize)
 app.use(configRequiredRoutes, populateSettings)
 app.use(filterOldRequests)
 
+app.use('/graphql', recordPing)
+graphQLServer.applyMiddleware({ app })
```
Comment (chaotixkilla) [inline lib/routes.js]: Does the order matter here? I'd personally prefer the routes to be grouped up together, but it's a nitpick

**Review by chaotixkilla — APPROVED** [review]

File: lib/plugins.js (hunk 1 above)
Comment (siiky) [reply lib/plugins.js:368]: Yeah, taking 5% off is multiplying by 0.95, not dividing by 1.05.

File: lib/routes.js (hunk 2 above)
Comment (siiky) [reply lib/routes.js]: Hmm not sure, I'll try.

Comment (siiky) [reply lib/routes.js]: Seems to work.

**Review by chaotixkilla — APPROVED** [review]

File: lib/graphql/resolvers.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      _.set('restartServices', !!pid && state.restartServicesMap?.[operatorId]?.[deviceId] === pid),
+    ))
+}
+
+
+module.exports = {
+  Query: {
+    staticConfig,
+    dynamicConfig
+  }
```
Comment (RafaelTaranto) [inline lib/graphql/resolvers.js]: I think I would rather have a single query that returns both static and dynamic config. If you organize that way it means that you can run pollQueries before and pass it down as a parameter.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Pull request merged**